src/api/BamConstants.h

```diff
@@ -43,6 +43,7 @@ const int BAM_ALIGNMENT_READ_2 = 0x0080;
 const int BAM_ALIGNMENT_SECONDARY = 0x0100;
 const int BAM_ALIGNMENT_QC_FAILED = 0x0200;
 const int BAM_ALIGNMENT_DUPLICATE = 0x0400;
+const int SUPPLEMENTARY           = 0x0800;  // Additional patch by tiddit package
```

Comment:
this is clearly not using the same naming scheme as the other constants. I've merged https://github.com/pezmaster31/bamtools/commit/afd7e94312b66fd73f9c3c9ee6f04c29059b2a1b and will close this PR, we should have a new release soon and then you should patch tiddit to use the canonical